courses/api.py

```diff
@@ -1361,32 +1391,31 @@ def get_verifiable_credentials_payload(certificate: BaseCertificate) -> dict:
         if certificate.issue_date
         else now_in_utc().strftime("%Y-%m-%dT%H:%M:%SZ")
     )
-    return {
+    activity_end_date = valid_from
+    payload = {
         "@context": [
             "https://www.w3.org/ns/credentials/v2",
             "https://purl.imsglobal.org/spec/ob/v3p0/context-3.0.3.json",
             "https://w3id.org/security/suites/ed25519-2020/v1",
         ],
-        # TODO: Need to figure out if this should use the same value as the certificate
         "id": f"urn:uuid:{certificate.uuid}",
         "type": ["VerifiableCredential", "OpenBadgeCredential"],
         "issuer": {
-            "id": "did:key:z6MkjoriXdbyWD25YXTed114F8hdJrLXQ567xxPHAUKxpKkS",  # TODO: replace with real DID
+            "id": f"did:key:{settings.VERIFIABLE_CREDENTIAL_DID}",
             "type": ["Profile"],
             "name": "MIT Learn",
             "image": {
-                # TODO: Needs to be hosted somewhere better
+                # TODO: replace w/ output of https://github.com/mitodl/hq/issues/9657 # noqa: TD002, FIX002
```

Comment:
via https://github.com/mitodl/hq/issues/9657#issuecomment-3676333541

this will be permantently available at https://learn.mit.edu/images/mit-red.png